Terminal.Gui/Input/Key.cs

```diff
@@ -971,4 +971,19 @@ out parsedInt
     public static Key F24 => new (KeyCode.F24);
 
     #endregion
+
+    private static Rune _shortcutDelimiter = new ('+');
+
+    /// <summary>Sets or gets the shortcut delimiter separator. The default is "+".</summary>
+    public static Rune ShortcutDelimiter
```

Comment:
We should rename this to "Separator". With this PR, this is now used for all `Key` usages and should be more generic.

We should also make this a CM property. It needs to be configurable.

On Windows the standard is +.

On Mac the standard is -.

Reply:
> We should rename this to "Separator". With this PR, this is now used for all `Key` usages and should be more generic.
> 
Done.

> We should also make this a CM property. It needs to be configurable.
> 
Done.

> On Windows the standard is +.
> 
Done.

> On Mac the standard is -.

I think this is irrelevant and I have no idea how to configure this on the `config.json` to achieve this.